x-pack/plugins/ml/public/ml_url_generator/ml_url_generator.test.ts

```diff
@@ -190,7 +192,9 @@ describe('MlUrlGenerator', () => {
             groupIds: ['group_1', 'group_2'],
           },
         });
-        expect(url).toBe('/app/ml/data_frame_analytics?mlManagement=(groupIds:!(group_1,group_2))');
+        expect(url).toBe(
+          "/app/ml/data_frame_analytics?_a=(data_frame_analytics:(queryText:'groups:(group_1%20or%20group_2)'))"
```

Comment:
We should let APM and Metrics team know that this is changing as I believe they may be using the old query param setup. cc @phillipb, @sqren